src/ScratchStorage.ts

```diff
@@ -182,15 +200,14 @@ class ScratchStorage {
      *   If the promise is rejected, there was an error on at least one asset source. HTTP 404 does not count as an
      *   error here, but (for example) HTTP 403 does.
      */
-    load (assetType, assetId, dataFormat) {
-        /** @type {Helper[]} */
+    load (assetType: AssetType, assetId: AssetId, dataFormat: DataFormat): Promise<_Asset | null> {
         const helpers = this._helpers.map(x => x.helper);
-        const errors = [];
+        const errors: unknown[] = [];
```

Comment:
I'm still a little embarrassed that I didn't use `Error` throughout :/

Reply:
I didn't check, to be honest, but having promise rejections be `unknown` in TS is standard practice since any rejection up the chain can be an arbitrary object and this can even be libraries. So even if we use Error all around our code, it still is hard to make sure it's an Error in every possible case.